### PR TITLE
Main Tasks list

### PR DESCRIPTION
**Pull request by alexpersian**
- Create the main tasks list view
- Connects the task tableview to the remote data source
- Allows for creating new tasks through the + button

Future additions:
- Ability to delete a task
- Ability to mark a task as "Complete"

### CODE REVIEW

File: Examples/TodoApp/TodoApp/src/tasks/TasksInjector.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
+//
```
Comment (andreacipriani) [inline Examples/TodoApp/TodoApp/src/tasks/TasksInjector.swift]: should we remove the boilerplate comments? anyway, we have to attach the proper license before merging

File: Examples/TodoApp/TodoApp/src/tasks/TasksViewController.swift [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        alertController.addAction(doneAction)
+
+        addTaskModal = alertController
+    }
+
+    // MARK: - Updates
+
+    private func updateTaskList() {
+        activityIndicator.startAnimating()
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```
Comment (andreacipriani) [inline Examples/TodoApp/TodoApp/src/tasks/TasksViewController.swift]: is this delay to also simulate "loading..."?

**Review by andreacipriani — APPROVED** [review]
Nice, LGTM!  I guess we can extract some things out of the VC when we'll work on the Mobius integration

File: Examples/TodoApp/TodoApp/src/tasks/TasksViewController.swift (hunk 2 above)
Comment (alexpersian) [reply Examples/TodoApp/TodoApp/src/tasks/TasksViewController.swift]: Whoops, I think this isn't needed any longer. Will remove.

**Pull request merged**